src/modules/controllers/task.controller.js

```diff
@@ -0,0 +1,32 @@
+const Task = require('../../db/models/task/index');
+
+module.exports.getAllTasks = (req, res, next) => {
```

Comment:
export is done at the end. All functions at once. That is 
module.exports { 
  getAllTasks,
  ...
}